### PR TITLE
draft a blog post about email-compat

### PR DESCRIPTION
**Pull request by r10s**
- the current image is only a placeholder to get the idea, we need some nicer things here

- the post will be linked from the device messages drafted at https://github.com/deltachat/deltachat-android/pull/1911

**please do not merge yet.**

### CODE REVIEW

File: _posts/2021-05-05-email-compat.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+- [Bots](https://delta.chat/en/2020-03-26-shining-some-light-on-bots) also benfit from several features -
+  eg. they also can anonymize users and can send HTML-mails.
+
+- **Speedup global search**
+
+- you can see the **status** (the footer) of each contact
+  in the contact's profile (the feature is available on Android, other systems will follow)
+
+
+## Is it for anyone?
```
Comment (missytake) [inline _posts/2021-05-05-email-compat.md]:
```suggestion
## Is this for everyone?
```

I think "anyone" has the nuance of "Ist es für irgendwen?" while we want to say "Ist das für alle?" (do we want to say that?)

**Review by missytake — APPROVED** [review]
Love it!

File: _posts/2021-05-05-email-compat.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+Delta Chat 1.20 supports **reading mailing lists.**
+
+Mailing lists are especially important as
+also "normal" mails may come as mailing lists -
+e.g. the mentioned notification from the parcel service
+is now displayed even if wrapped into a mailing list.
+
+As sender on mailing lists may be anonymized by the mailing list software,
+they are are marked by a leading tilde as **~Alice**.
```
Comment (adbenitez) [inline _posts/2021-05-05-email-compat.md]:
```suggestion
they are marked by a leading tilde as **~Alice**.
```

File: _posts/2021-05-05-email-compat.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+So, in general, classic e-mail works since the first day:
+
+- You can **chat with any e-mail address,**
+  there is no need that the recipient also uses Delta Chat.
+  The recipient just receives a classic e-mail then
+  (not only a link as some other apps do).
+
+- To **receive classic e-mails** in addition to chat messages,
+  you need to enable **Settings / Show classic mail.**
```
Comment (adbenitez) [inline _posts/2021-05-05-email-compat.md]:
```suggestion
  you need to enable **Settings / Chats and media / Show classic emails.**
```

Comment (adbenitez) [reply _posts/2021-05-05-email-compat.md]: mmm.... don't know how it is on iOS and Desktop, one more reason to sync menu structure across the different platforms

Comment (r10s) [reply _posts/2021-05-05-email-compat.md]: it is correct for ios, and desktop settings are flat anyway.

**Comment by r10s** [comment]
k, 1.20 seems to be accepted for gplay, rolling out to 10% currently, i think, we should merge that soon.

**Pull request merged**